elyra/metadata/schemas/kfp.json

```
@@ -120,6 +120,16 @@
             "ui:placeholder": "https://your-cos-service:port"
           }
         },
+        "cos_public_endpoint": {
```

Nit: To remain consistent with the existing naming (`api_endpoint` / `public_api_endpoint`) we should name this property 

```suggestion
        "public_cos_endpoint": {
```

---

i think this names makes more sense with `cos` before, since this is the context, but changed 😃